.eslintrc

```diff
@@ -25,7 +25,9 @@
           "props": false
         }
       }
-    ]
+    ],
+    "unicorn/no-null": "off",
```

Comment:
That's a new one on me :+1:

Reply:
:D there's a good case made in the linked articles on [the rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-null.md), but unfortunately I don't have the bandwidth right now to confirm that (e.g.) pdfmake supports both undefined and null in the places we're currently passing null values. 

I don't agree with the no-reduce rule though - IMO it has its place :).